app/javascript/controllers/new_message_controller.js

```diff
@@ -27,8 +27,10 @@ export default class extends Controller {
   }
 
   submitForm() {
-    if (this.cleanInputValue.length > 0)
-      this.element.submit()
+    if (this.cleanInputValue.length > 0) {
+      this.element.requestSubmit()
```

Comment:
We definitely need to be using requestSubmit, without this morphing doesn't work.